cloud_provider_requirements.md

```diff
@@ -0,0 +1,17 @@
+# Cloud Provider Requirements
+
+The Spinnaker project has an interest in ensuring that cloud provider integrations are of high quality and actively
+maintained.  In order to achieve this goal, we have established criteria for a new cloud provider integration to be
+accepted into the project.
+
+In order for an integration to remain in the project, the following conditions must be met:
+* The cloud provider integration is actively supported by an organization with a significant stake in the integration
+  * This could be the cloud provider itself, or a user of that cloud provider
+* Major bugs and issues with the integration are addressed in a timely manner
+* The maintainers handle routine maintenance tasks for the integration, such as:
+  * Keeping it compatible with updates to the cloud provider itself
+  * Making any cloud-provider specific changes in response to global architecture changes in Spinnaker
```

Comment:
Would like to add another requirement that a SIG is formed & kept active for the cloud provider.